containerized_ml_models/api/api.py

```diff
@@ -0,0 +1,24 @@
+from fastapi import FastAPI, Request
+from pickle import load
+import numpy as np
+from uvicorn import run
+
+app = FastAPI()
+model = load(open("model.pkl", "rb"))
+
+
+@app.get("/ping")
+def ping():
+    return {"ping": "success"}
+
+
+@app.post("/predict")
+def predict():
+    request_data = Request.get_json()
+    data = np.array([request_data["data"]])
+    prediction = model.predict(data)
+    return {"Result": prediction.tolist()}
+
+
+if __name__ == "__main__":
+    run(app, host="0.0.0.0", port=8000)
```

Comment:
No newline at the end of the file 😞

Reply:
There is a newline here but github just doesnt show it, its indicated by this symbol: 
![image](https://user-images.githubusercontent.com/77837063/201477798-87d76534-da32-4b3b-a9a5-77142b7013fd.png)